server/modelaggregator.edl

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+    trusted {
+        public void enclave_modelaggregator([in, count=length] unsigned char*** encrypted_accumulator,
+            [in, count=length] uint32_t* decrypted_accumulator_lengths,
+            size_t length,
+            [user_check] unsigned char** encrypted_old_params,
+            uint32_t decrypted_old_params_length,
+            [out] unsigned char*** encrypted_new_params_ptr);
+    };
+
+    untrusted {
```

Comment:
This function does not need to be here, remove the entire `untrusted` block.
The `untrusted` block is used to identify ocalls (i.e. functions in the host code called by the enclave code). Since the `host_modelaggregator` function is not called by the enclave, it doesn't need to be here.